CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+## Unreleased changes
+
```

Comment:
Perhaps we  can add a line like `[Full Changelog](https://github.com/vcr/vcr/compare/v6.0.0...master)`, here, to make it even more complete?

Not a blocker, totally, not a blocker.